frontend/components/layout/Header.vue

```diff
@@ -1,8 +1,23 @@
 <template>
     <div>
+        <!-- Sky banner -->
+        <div class="w-full p-2 text-center bg-banner-red">
+            <h1 class="text-white text-xl">
+                MakerDAO is now Sky — the next evolution of DeFi. Explore
+                <a class="underline" href="https://sky.money/" target="_blank">Sky.money</a> and get rewarded for
+                saving without giving up control.
+            </h1>
+        </div>
         <StagingBanner v-if="stagingBannerUrl" :url="stagingBannerUrl" />
-        <ProductionBannerElectron v-if="productionBannerUrl" :url="productionBannerUrl" />
-        <header class="bg-primary dark:bg-primary-dark">
+        <header
+            class="
+                bg-gradient-to-r
```

Comment:
Nit: let's maybe try using either just semi-transparent white/gray color here or top-to-bottom gradient. I don't see in the Brand Guideline any gradient that is left to right.